### DISCUSSION TITLE
Example that prevent showing Vue template

### DISCUSSION THREAD

**Discussion by jcubic** [vuejs/petite-vue#96] [Ideas]
If you have fast internet you may not see this, but even on no so slow network you can see the template for a split of a second.

Maybe it's worth adding to the readme, first thing that came into my mind when tried the application. I'm not sure if the same thing happens in Vue and everybody knows this.

```html
<div class="cloak">
    <div v-scope="{ count: 0 }">
        {{ count }}
        <button @click="count++">inc</button>
    </div>
</div>
```

```css
.cloak {
    display: none;
}
```

```javsscript
PetiteVue.createApp().mount();
document.querySelectorAll('.cloak').forEach(node => {
    node.classList.remove('cloak');
});
```

https://codepen.io/jcubic/pen/PoKZEKz

**Comment by TomS-** [comment]
`v-cloak` is already a feature. Just add `[v-cloak] { display: none; }`.
